examples/simple_resource.py

```diff
@@ -11,15 +11,15 @@
 from lbz.router import add_route
 
 
-class HelloWorld(Resource):
+class HelloWorldExample(Resource):
```

Comment:
What was the problem with the previous name? It is used also in `README.md`, so I would change that in both places, if we really need 😉

Reply:
pylint.

Reply:
But `pylint` what? 🤔  Do you mean code duplication? And as I see caused by imports...

We should disable this warning for time being as we did in other places 😉